drivers/ieee802154/ieee802154_rf2xx_iface.c

```diff
@@ -7,7 +7,11 @@
  */
 
 #define LOG_MODULE_NAME ieee802154_rf2xx_iface
+#if defined(CONFIG_IEEE802154_DRIVER_LOG_LEVEL)
```

Comment:
Why this ifdef?
#define LOG_LEVEL CONFIG_IEEE802154_DRIVER_LOG_LEVEL  
should be sufficient.

Reply:
Rollback